### PR TITLE
Add ci cd scripts

### PR DESCRIPTION
**Pull request by Hyperkid123**
part of: https://issues.redhat.com/browse/RHCLOUD-16636

cc @BlakeHolifield can you take a peek at the scripts? I am wondering if we should use podman instead of docker.

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+source unit_test.sh
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="quickstarts"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="quickstarts"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/quickstarts"
+
+IQE_PLUGINS="e2e"
```
Comment (BlakeHolifield) [inline pr_check.sh:13]: I think this is good? Generally, we use the app's name and that has to be configured in the smoke test framework that iqe uses. Not sure.

File: pr_check.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+IQE_CJI_TIMEOUT="30m"
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
+source $CICD_ROOT/build.sh
+source $CICD_ROOT/deploy_ephemeral_env.sh
+
+# Deploy HBI app required for the smoke tests
```
Comment (BlakeHolifield) [inline pr_check.sh:25]: Is HBI a dependency of quickstarts? Also, you aren't smoke testing at all yet.

**Review by BlakeHolifield — APPROVED** [review]
This should be fine, and is generally a pretty iterative process. This is a good first start and should work.

**Comment by BlakeHolifield** [comment]
With regards to podman v docker, it depends on what jenkins node these jobs run on. I'd say leave it as `docker` for now and we'll move to podman when we get this rolling.

File: pr_check.sh (hunk 2 above)
Comment (Hyperkid123) [reply pr_check.sh:25]: Definitely not. I forgot to remove it while looking at the examples in the docs.

**Comment by Hyperkid123** [comment]
@BlakeHolifield Thanks for the comments.

**Pull request merged**